config/routes.rb

```diff
@@ -1,4 +1,17 @@
 Rails.application.routes.draw do
+  devise_for :users, defaults: { format: :json },
+             controllers: {
+                 sessions: 'users/sessions',
+                 registrations: 'users/registrations'
+             }
+  # get '/member-data', to: 'members#show'
+
+  
+  resources :addresses
+  resources :doctors
+  resources :users do
+    resources :reservations
+  end
```

Comment:
- [ ] I suggest that you specify the routes you will need with `only`, since this creates a lot of routes you won't be using.